installation/README.md

```diff
@@ -1,3 +1,95 @@
+- Certificate for Remote Environments
+- IP address for Kyma Ingress
+- IP address for Remote Environments Ingress
+
+### 0. Prerequisites
+
+Kubernetes API Server is configured in the following manner:
+
+```
+"apiServerConfig": {
+    "--enable-admission-plugins": "Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,DefaultStorageClass,ResourceQuota,PodPreset",
+    "--runtime-config": "batch/v2alpha1=true,settings.k8s.io/v1alpha1=true,admissionregistration.k8s.io/v1alpha1=true",
+    "--cors-allowed-origins": ".*",
+    "--feature-gates": "ReadOnlyAPIDataVolumes=false",
+    "--oidc-issuer-url": "https://dex.__DOMAIN_NAME__",
+    "--oidc-client-id": "kyma-client",
+    "--oidc-username-claim": "email", 
+    "--oidc-groups-claim": "groups"
+},
+"kubeletConfig": {
+    "--feature-gates": "ReadOnlyAPIDataVolumes=false",
+    "--authentication-token-webhook": "true",
+    "--authorization-mode": "Webhook"
+}
+```
+
+### 1. `kyma-installer` namespace
+
+Create the `kyma-installer` namespace executing following command:
+
+```
+kubectl create ns kyma-installer
+```
+
+### 2. The `installer-config.yaml.tpl` template
+
+The Kyma installation process requires installation data specified in the `installer-config.yaml` file. Copy the `installer-config.yaml.tpl` template, rename it to `installer-config.yaml`, and fill in the placeholders.
+
+- `__TLS_CERT__` Placeholder for TLS certificate
+- `__TLS_KEY__` Placeholder for TLS certificate key
+- `__REMOTE_ENV_CA__` Placeholder for Remote Environments CA
+- `__REMOTE_ENV_CA_KEY__` Placeholder for Remote Environments CA key
+- `__EXTERNAL_IP_ADDRESS__` Placeholder for IP address for Kyma Ingress
+- `__DOMAIN__` Placeholder for the domain name
+- `__REMOTE_ENV_IP__` Placeholder for IP address for Remote Environments Ingress
+- `__K8S_APISERVER_URL__` Placeholder for API server's URL
+- `__K8S_APISERVER_CA__` Placeholder for your API Server CA
+- `__ADMIN_GROUP__` Placeholder for additional admin group (optional, may be left blank)
+- `__ENABLE_ETCD_BACKUP_OPERATOR__` Placeholder for enabling/disabling `etcd` backup operator ("true", "false")
+- `__ECTD_BACKUP_ABS_CONTAINER_NAME__` Placeholder for Azure Blob Storage name for `etcd` backups, may be left blank when backup operator is disabled
+
+Once all the required placeholders are filled in, run the following command to provide the cluster with the installation data:
+
+```
+kubectl apply -f installer-config.yaml
+```
+
+### 3. Deploy the `Installer` component
+
+To deploy the `Installer` component on your cluster, run the following command:
+
+```
+kubectl apply -f installation/resources/installer.yaml -n kyma-installer
+```
+
+### 4. Trigger the installation
+
+To trigger the installation of Kyma, you will need a Custom Resource file. Copy the `installer-cr.yaml.tpl` file, rename it to `installer-cr.yaml`, and fill in the placeholders:
```

Comment:
you will need -> `you need `
fill in the placeholders: -> `fill in these placeholder values:`